extensions/extensions-data-lineage/extensions-data-lineage-http-consumer-service/pom.xml

```diff
@@ -139,26 +139,6 @@
                     <skipOriginalJarRename>true</skipOriginalJarRename>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
I: As mentioned above, this is unnecessary, so will be cleaning up as part of this PR.